Signal/src/view controllers/NewGroupViewController.m

```diff
@@ -403,4 +403,10 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField {
     return NO;
 }
 
+#pragma mark - UIScrollViewDelegate
+
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView {
```

Comment:
nit: clang-format.

I *remembered* something! `git-clang-format` fails to detect changes if the changed file has a space in it's path (../view&lt;space&gt;controllers/..).

If you're feeling brave you can apply this patch to git-clang-format (which is just a python script):
```
diff --git a/git-clang-format b/git-clang-format
index 74fd451..ca276e6 100755
--- a/git-clang-format
+++ b/git-clang-format
@@ -302,7 +302,7 @@ def extract_lines(patch_file):
   for line in patch_file:
     match = re.search(r'^\+\+\+\ [^/]+/(.*)', line)
     if match:
-      filename = match.group(1).rstrip('\r\n')
+      filename = match.group(1).rstrip('\r\n\t')
     match = re.search(r'^@@ -[0-9,]+ \+(\d+)(,(\d+))?', line)
     if match:
       start_line = int(match.group(1))
(11:09:28) ± [mkirk@ash.local] /usr/local/Cella
```

Or if you're feeling *really* bold you can rename `view controllers` to `ViewControllers`.